### PR TITLE
[docs] Call out platform limitations in our docs.

### PR DESCRIPTION
**Pull request by surfnerd**
### Proposed change(s)
Call out platform limitation in our docs

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [x] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: com.unity.ml-agents/Documentation~/com.unity.ml-agents.md [hunk 1]
```diff
@@ -58,6 +58,9 @@ This version of the Unity ML-Agents package is compatible with the following ver
 
 ## Known limitations
 
+### Training
+Training is limited to the Unity Editor and Standalone builds on Windows, MacOS, and Linux.  Your environment will default to inference mode if training is not supported or is not currenlty running.
```
Comment (mmattar) [inline com.unity.ml-agents/Documentation~/com.unity.ml-agents.md:62]: currenlty --> currently.

File: com.unity.ml-agents/Documentation~/com.unity.ml-agents.md [hunk 2]
```diff
@@ -58,6 +58,9 @@ This version of the Unity ML-Agents package is compatible with the following ver
 
 ## Known limitations
 
+### Training
```
Comment (mmattar) [inline com.unity.ml-agents/Documentation~/com.unity.ml-agents.md:61]: Do we need to say anything about Inference/Barracuda?

Comment (surfnerd) [reply com.unity.ml-agents/Documentation~/com.unity.ml-agents.md:61]: They don't call out any platform limitations in their docs so i'm not sure what to point to.

Comment (mmattar) [reply com.unity.ml-agents/Documentation~/com.unity.ml-agents.md:61]: Sounds good.

**Pull request merged**